dewy/common/collection_embeddings.py

```diff
@@ -10,7 +11,20 @@
 from dewy.collection.models import DistanceMetric
 from dewy.config import Config
 
-from .extract import extract
+from .extract import extract_content, extract_url
+
+
+@dataclasses.dataclass
+class IngestContent:
+    filename: Optional[str]
+    content_type: Optional[str]
```

Comment:
I don't see this being used anywhere, and we seem to be pulling mime type from the file - is this needed?

Reply:
Pulled it in so we don't lose it and can plumb it through / log it / use it if helpful.

Reply:
Moved to #84.